src/creatures/players/player.cpp

```diff
@@ -5189,14 +5189,14 @@ uint16_t Player::getSkillLevel(skills_t skill) const {
 }
 
 bool Player::isPremium() const {
-	if (g_configManager().getBoolean(FREE_PREMIUM) || hasFlag(PlayerFlags_t::IsAlwaysPremium)) {
+	if ((!g_configManager().getBoolean(VIP_SYSTEM_ENABLED) && g_configManager().getBoolean(FREE_PREMIUM)) || hasFlag(PlayerFlags_t::IsAlwaysPremium)) {
```

Comment:
I think this breaks my usage of the VIP system -- i want players to always be premium regardless of their vip status

Reply:
If I have vip system enabled, the config for free premium isn't available.